### ISSUE TITLE
OC Web showing blank screen in Chrome Browser

### ISSUE THREAD

**Issue by GrendelWWU** [owncloud/web#5961]
### Steps to reproduce
1. login to account
2. click on new design

### Expected behaviour
showing OC Web Interface

### Actual behaviour
blank screen appears

### Environment general
**Operating system**:
Server: Red Hat Enterprise Linux Server release 7.3 (Maipo)
Server: Kernel: Linux 3.10.0-957.el7.x86_64
Client: Chrome Browser: Version 84.0.4147.89 (Offizieller Build) (64-Bit)
Client: Ubuntu 20.04 LTS, Mate, Linux 5.4.0-89-generic

**Backend (ownCloud Core or Infinite Scale)**:
oC 10.8.0.4

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
LDAP

### Environment ownCloud Infinite Scale
**Version:** 10.8.0

**Updated from an older Infinite Scale or fresh install:**
oC updated from 10.6.0, Web App updated from 4.2 (afair)

**Where did you install Infinite Scale from:**

### Environment ownCloud Core
**Web server:** Apache httpd.x86_64                    2.4.6-97.el7_9.1             @rhel-7-server-rpms

**Database:** mySQL/MariaDB cluster

**PHP version:**
php.x86_64                      7.4.25-1.el7.remi            @remi-php74 

**Version:** (see ownCloud admin page)

**Updated from an older ownCloud or fresh install:**

**Where did you install ownCloud from:**
customer site

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results into https://gist.github.com/ and put the link here.
```

**The content of config/config.php:**

```
Log in to the web-UI with an administrator account and click on
'admin' -> 'Generate Config Report' -> 'Download ownCloud config report'
This report includes the config.php settings, the list of activated apps
and other details in a well-sanitized form.

or 

If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

*ATTENTION:* Do not post your config.php file in public as is. Please use one of the above
methods whenever possible. Both, the generated reports from the web-ui and from occ config:list
consistently remove sensitive data. You still may want to review the report before sending.
If done manually then it is critical for your own privacy to diligently
remove *all* hostnames, passwords, usernames, salts and other credentials before posting.
You should assume that attackers find such information and will use them against your systems.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder.
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by kulmann** [comment]
Release 4.4.0 is already out since last week. :-) We'll work on having it in 4.5.0, which will be released in about 2 weeks.

**Comment by GrendelWWU** [comment]
I installed 4.5 of web-app last week. My Chrome browser is (I guess) the latest version. 84.0.4147.89 (64bit).
When clicking on _New Design_ it'll show a blank screen. I removed the cookies for this site. On another Ubuntu Linux (Upgraded to 20.04 LTS) machine, it works. I really don't know, what is happening there. I will install a new 20.04 in a VM later. Maybe there's just something wrong here?!
[error.txt](https://github.com/owncloud/web/files/7578648/error.txt)
But.... JavaScript isn't disabled in my settings...
